x-pack/plugins/security_solution/server/lib/telemetry/receiver.ts

```diff
@@ -254,7 +343,7 @@ export class TelemetryReceiver {
     }
 
     // Ensure list is created if it does not exist
-    await this.exceptionListClient.createTrustedAppsList();
+    await this.exceptionListClient.createEndpointList();
```

Comment:
NOTE: This looks like it should have been `createEndpointList` since this function is `fetchEndpointList`